src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -481,3 +481,134 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
         except Exception as e:  # pylint: disable=broad-exception-caught
             logger.error(f"Couldn't fetch information for table {full_name} : {e}")
             return None
+
+
+class FasterTableScanCrawler(CrawlerBase):
```

Comment:
add `use_experimental_tables_scan` to `WorkspaceConfig` and modify the `WorkflowsDeployment` to invoke that from if that config value is active. Add an integration test with that config active.

```
class Assessment(Workflow):
    ...
    @job_task(notebook="hive_metastore/tables.scala")
    def crawl_tables(self, ctx: RuntimeContext):
```

Reply:
Will add this in the next PR to utilize the new FastTableScanCrawler class with integration tests